src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Program.cs

```diff
@@ -5,6 +5,7 @@
 using AWS.Deploy.Recipes.CDK.Common;
 using AspNetAppElasticBeanstalkLinux.Configurations;
 using Microsoft.Extensions.Configuration;
+using Environment = Amazon.CDK.Environment;

```

Comment:
why these extra imports?

Reply:
After updating, ambiguous references started to show up. That's why these are needed.